bin/config-rev

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+var CONFIG_FILENAME = 'deploy-config.js';
+var DEFAULT_CONFIG_ABBREV = 7;
+
+var path = require('path');
+var config = require(path.join(process.cwd(), CONFIG_FILENAME))
```

Comment:
There are more places where the config could be stored. See https://github.com/productboard/webpack-deploy/pull/28/files#diff-1bf37fe770044c540521fd30ab26dec4R19
We could just export a function to get the config file.